first-test/java-testng/src/test/java/FirstWebDriverTestJavaTestNGTest.java

```diff
@@ -9,8 +10,10 @@
 
     @Test
     public void checkPHPTravelsPageTitle() throws MalformedURLException {
-        // Create a new instance of the Firefox driver
-        WebDriver driver = new FirefoxDriver();
+        // Create a new instance of the Chrome driver
+        WebDriver driver = new ChromeDriver();
```

Comment:
Can you please remove the ChromeDriver usage? Otherwise I would need to tell the participants of the workshop to install another tool in their laptops.
Please switch it to use docker-selenium, or just remove the code and I can also do it :)
After this, I could merge the PR.